app/controllers/admin/creator_settings_controller.rb

```diff
@@ -12,13 +13,10 @@ def create
         ::Settings::UserExperience.primary_brand_color_hex = settings_params[:primary_brand_color_hex]
         ::Settings::Authentication.invite_only_mode = settings_params[:invite_only]
         ::Settings::UserExperience.public = settings_params[:public]
+        settings_params[:checked_code_of_conduct]
+        settings_params[:checked_terms_and_conditions]
```

Comment:
Let's remove these two lines of code from here.

Initially I didnt notice that these are in an `ActiveRecord::Base.transaction` which allows for protective blocks where SQL statements are only permanent if they can all succeed as one atomic action.

All the assignments in the block are assignments to something other than the current user. Hence this should not be in this block. Rubucop is rightly complaining that the assignment is useless.

Reply:
Ahhh, that makes perfect sense now! I've removed these lines of code.